### DISCUSSION TITLE
How to use solar charge without depleting the solar battery

### DISCUSSION THREAD

**Discussion by theflexa1** [serkri/SmartEVSE-3#262] [General]
Hi to everyone.

I´m ussing the Smart EVSE and I have the problem that I don´t know how to configure this...

It´s possible to have some heavy help with this? I´ve tryed other chargers but this one is the only that seems to solve this problem. My kwnolegde in this type of configurations is really "noob".

Thank you all.

**Comment by dingo35** [comment]
General manual:
https://github.com/dingo35/SmartEVSE-3.5
(see links bottom page)

Config of battery API:
https://github.com/dingo35/SmartEVSE-3.5/blob/master/docs/configuration.md#home-battery-integration

**Reply by theflexa1** [reply]
I saw this but this is only info. Nothing else. There is a manual o a procidure tu make this work... Sorry, i´m a bit lost.

This info you linked, it´s the same I saw :(
